VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitWebviewFragment.java

```diff
@@ -71,6 +81,10 @@ public void onDestroy(){
         super.onDestroy();
     }
 
+    public void onEvent(NetworkConnectivityChangeEvent event){
+        setFragmentState(!NetworkUtil.isConnected(getContext()));
```

Comment:
`registerSticky()` can be a convenience method to avoid duplicating logic if we use it properly (e.g. unregister and register from the `setUserVisibleHint()` callback as well based on the visibility). In any case, I think we should handle it proactively, as the web view seems to take a long time to timeout.